starter-code/src/functions-and-arrays.js

```diff
@@ -11,10 +19,27 @@ var words = [
   'crackpot'
 ];
 
+function findLongestWord(wordsArray){
+var longestWord = 0
+var wordsArray = [];
+
+for(var i = 0; i < wordsArray.length; i++){
+   if(wordsArray[i].length >= longestWord){
+     longestWord = wordsArray[i].length;
+   }
+ }
+ return longestWord;
+}
+findLongestWord("foo");
+
 // Calculating a Sum
 
 var numbers = [6, 12, 1, 18, 13, 16, 2, 1, 8, 10];
 
+function sumArray(){
```

Comment:
Empty Function? You never completed this step. Here is the solution for this step, so you can check it out and make sense of how it works:

```js
// Calculating a Sum
function sumArray(array) {
  var sum = 0;
  for (var x = 0; x < array.length; x++) {
    sum += array[x];
  }
  return sum;
}

var numbers = [6, 12, 1, 18, 13, 16, 2, 1, 8, 10];
var total = sumArray(numbers);
console.log(total);
```